docs/src/content/docs/concepts/http-signatures.md

```diff
@@ -0,0 +1,84 @@
+# HTTP Signatures
+
+The Open Payments API uses HTTP signatures to secure the integrity of messages between sender, receiver, and third-party initiating payment systems. HTTP signatures are also part of the process that controls access to protected resources that orchestrate payment transactions.
+
+## Purpose
+
+Cryptographic digital signatures make it possible to secure HTTP messages by verifying the:
+
+- **Authenticity** of a system that requests permissions for access to specific resources.
+- **Integrity** of messages that are exchanged between Open Payments API enabled systems. This guards against the risk of message tampering, and can apply to the whole message or parts of the message.
+
+In this context, an authorizing server controls API access by granting particular permissions to a specific client.
+
+## Understanding HTTP Signatures
+
+### Specification
+
+<!-- NOTE: the latest published spec. version, as at Oct 2023, is 19. How do we approach updating our spec. refs? Do we first analyse the spec, to ensure that our implementation is in-line with the latest version?
+-->
+The Open Payments API adheres to the HTTP Message Signature specification drafted by the HTTP workgroup of the Internet Engineering Task Force (IETF).
+The referenced specification is published here: [httpbis-message-signature-16](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-message-signatures-16).
```

Comment:
I can't answer the question, the tech team will need to advise, but if you remove the version number from the URL (so, ...httpbis-message-sigatures), it'll link to the most recent version. That's probably what we want to do unless we're told otherwise.